app/services/foreman/plugin.rb

```diff
@@ -243,6 +243,27 @@ def role(name, permissions)
       end
     end
 
+    # Add plugin permissions to core's Manager and Viewer roles
+    # Usage:
+    # add_permissions_to_core_roles ['MyPlugin::FirstResource', 'MyPlugin::SecondResource'], :except => [:skip_this_permission]
+    def add_resource_permissions_to_core(resources, opts = {})
+      Role.transaction do
+        add_viewer_roles(resources, opts.dup)
+        add_manager_roles(resources, opts.dup)
```

Comment:
These method names are a little misleading as we aren't adding manager roles, but rather adding permissions to the single manager or viewer role.